### PR TITLE
Add extension properties to nodes api

### PR DESCRIPTION
**Pull request by eshaparmar**

### CODE REVIEW

**Comment by eshaparmar** [comment]
@vrdmr

File: src/ResourceManagement/Automation/AutomationManagement/Generated/DscNodeConfigurationOperations.cs [hunk 1]
```diff
@@ -236,7 +236,7 @@ public AutomationManagementClient Client
                     configurationValue["name"] = parameters.Configuration.Name;
                 }
```
Comment (vrdmr) [inline src/ResourceManagement/Automation/AutomationManagement/Generated/DscNodeConfigurationOperations.cs]: Let me check this. My assumption was that this change was pushed to both Azure and AA forks.

File: src/ResourceManagement/Automation/AutomationManagement/Generated/HybridRunbookWorkerGroupOperations.cs [hunk 2]
```diff
@@ -413,11 +413,11 @@ public AutomationManagementClient Client
...
                                     }
                                     
-                                    JToken lastSeenValue = hybridRunbookWorkersValue["lastSeen"];
-                                    if (lastSeenValue != null && lastSeenValue.Type != JTokenType.Null)
+                                    JToken lastSeenDateTimeValue = hybridRunbookWorkersValue["lastSeenDateTime"];
+                                    if (lastSeenDateTimeValue != null && lastSeenDateTimeValue.Type != JTokenType.Null)
                                     {
-                                        DateTimeOffset lastSeenInstance = ((DateTimeOffset)lastSeenValue);
-                                        hybridRunbookWorkerInstance.LastSeenDateTime = lastSeenInstance;
+                                        DateTimeOffset lastSeenDateTimeInstance = ((DateTimeOffset)lastSeenDateTimeValue);
```
Comment (vrdmr) [inline src/ResourceManagement/Automation/AutomationManagement/Generated/HybridRunbookWorkerGroupOperations.cs]: It looks like there are some commits which which should be in the master are not reflecting in it. Could you do a rebase and push the branch?

File: src/ResourceManagement/Automation/AutomationManagement/Generated/Models/DscCompilationJobCreateProperties.cs [hunk 3]
```diff
@@ -43,16 +43,16 @@ public DscConfigurationAssociationProperty Configuration
             set { this._configuration = value; }
         }
         
-        private bool _incrementBuildVersion;
+        private bool _incrementNodeConfigurationBuild;
```
Comment (vrdmr) [inline src/ResourceManagement/Automation/AutomationManagement/Generated/Models/DscCompilationJobCreateProperties.cs]: Same as the first comment. I'll take a look at this file.

File: src/ResourceManagement/Automation/AutomationManagement/Generated/Models/DscNode.cs [hunk 4]
```diff
@@ -41,6 +43,17 @@ public Guid AccountId
             set { this._accountId = value; }
         }
         
+        private IList<DscNodeExtensionHandlerAssociationProperty> _extensionHandler;
```
Comment (vrdmr) [inline src/ResourceManagement/Automation/AutomationManagement/Generated/Models/DscNode.cs:46]: You only require change in 3 files to go. Right? DscNodeOperations.cs, DscNode.cs, DscNodeExtensionHandlerAssociationProperty.cs. Right?

Comment (eshaparmar) [reply src/ResourceManagement/Automation/AutomationManagement/Generated/Models/DscNode.cs:46]: yes just these 3

**Comment by vrdmr** [comment]
@eshaparmar Corrected the state of the project. Please check the files in the PR and let me know if they are correct. Doing a git pull on master and rebasing should correct it on your end as well.

**Comment by eshaparmar** [comment]
Code reviewed in CodeFlow (http://codeflow/Client/CodeFlow2010.application?server=http://codeflow/Services/DiscoveryService.svc&review=esparmar-d623253919224dafae2fde7d546b0966)

**Review by vrdmr — APPROVED** [review]

**Pull request merged**